cmd/kubeadm/app/cmd/upgrade/node.go

```diff
@@ -94,6 +108,55 @@ func NewCmdUpgradeNodeConfig() *cobra.Command {
 	return cmd
 }
 
+// NewCmdUpgradeControlPlane returns the cobra.Command for upgrading the controlplane instance on this node
+func NewCmdUpgradeControlPlane() *cobra.Command {
+
+	flags := &controlplaneUpgradeFlags{
+		kubeConfigPath:   constants.GetKubeletKubeConfigPath(),
+		advertiseAddress: "",
+		dryRun:           false,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "experimental-control-plane",
```

Comment:
Why would we call out the upgrade path for a HA control plan node differently than the master?  I think we should detect from on cluster config and just roll with the upgrade.?.?.?

Reply:
That's fair feedback. I think this keeps the code simpler at the expense of the user. For an experimental thing I think that's fine. Once it becomes not experimental any more I'd fully expect this code to merge with the existing upgrade code. I see your point though and either we do extra work now to integrate it or integrate it later.

Reply:
I'm fine with calling this good enough for now, so long as there is a //TODO plus issue opened.